sos/utilities.py

```diff
@@ -325,11 +350,52 @@ def _check_poller(proc):
                 'truncated': reader.is_full
             }
     except OSError as e:
-        if to_file:
-            _output.close()
         if e.errno == errno.ENOENT:
             return {'status': 127, 'output': "", 'truncated': ''}
         raise e
+    finally:
+        if hasattr(_output, 'close'):
+            _output.close()
+
+
+def tac_logs(f_src, f_dst, drop_last_log=False):
+    """Python implementation of the tac utility with support
+    for multiline logs (starting with space). It is intended
+    to reverse the output of 'journalctl --reverse'.
+    """
+    NEWLINE_B = b'\n'
+    NEWLINE_I = 10
+    SPACE_I = 32
+    f_src.flush()
```

Comment:
Nice one!

Reply:
Added a comment to make it explicit why it's mandatory to flush here